packages/core/src/util/normalizeConfig.js

```diff
@@ -674,3 +678,50 @@ function normalizeSingleValue(configValue, defaultValue, configPath, envVarKey)
   }
   return configValue;
 }
+/**
+ * @param {InstanaConfig} config
+ */
+function normalizeIgnoreEndpoints(config) {
+  if (!config.tracing.ignoreEndpoints) {
+    config.tracing.ignoreEndpoints = {};
+  }
+
+  const ignoreEndpoints = config.tracing.ignoreEndpoints;
+
+  if (typeof ignoreEndpoints !== 'object' || Array.isArray(ignoreEndpoints)) {
+    logger.warn(
+      `Invalid tracing.ignoreEndpoints configuration. Expected an object, but received: ${JSON.stringify(
+        ignoreEndpoints
+      )}`
+    );
+    config.tracing.ignoreEndpoints = {};
+    return;
+  }
+
+  if (Object.keys(ignoreEndpoints).length) {
+    for (const [service, endpoints] of Object.entries(ignoreEndpoints)) {
+      const normalizedService = service.toLowerCase();
+
+      if (!Array.isArray(endpoints)) {
+        logger.warn(
+          `Invalid configuration for ${normalizedService}: tracing.ignoreEndpoints.${normalizedService} is not an array. The value will be ignored: ${JSON.stringify(
+            endpoints
+          )}`
+        );
+        config.tracing.ignoreEndpoints[normalizedService] = null; // Set to null for invalid configuration
+      } else {
+        config.tracing.ignoreEndpoints[normalizedService] = endpoints.map(endpoint => endpoint?.toLowerCase());
+      }
+    }
+  } else if (process.env.INSTANA_IGNORE_ENDPOINTS) {
+    // The environment variable name and its format are still under discussion.
+    // It is currently private and will not be documented or publicly shared.
+    try {
+      config.tracing.ignoreEndpoints = JSON.parse(process.env.INSTANA_IGNORE_ENDPOINTS);
+    } catch (error) {
+      logger.warn(
+        `Failed to parse INSTANA_IGNORE_ENDPOINTS: ${process.env.INSTANA_IGNORE_ENDPOINTS}. Error: ${error.message}`
+      );
+    }
```

Comment:
qs: Did you decide against the debug log to print values of `config.tracing.ignoreEndpoints`? Sorry I can't find the original conversation. GH hided it somewhere

Reply:
added log, discussed offline.